aten/src/ATen/native/cpu/batch_norm_kernel.cpp

```diff
@@ -244,64 +244,139 @@ batch_norm_cpu_collect_stats_channels_last_impl(
   // Normal size of C should fit in L1, otherwise consider blocking on C.
```

Comment:
I am not sure of how tiling/blocking on `C` could be useful for the original two-path reduction. In the first-path, `buffer_data` is updated in each iteration within each thread, and there is no data reuse?